src/firefighter/slack/messages/slack_messages.py

```diff
@@ -814,3 +814,61 @@ def get_blocks(self) -> list[Block]:
 
     def get_text(self) -> str:
         return f"Incident #{self.incident.id} might not need an incident channel, as it is {self.incident.priority.name}."
+
+
+class SlackMessageChannelReminderPostMortem(SlackMessageSurface):
+    id = "ff_reminder_post_mortem"
+
+    def __init__(self, incident: Incident):
+        self.incident = incident
+        super().__init__()
+
+    def get_text(self) -> str:
+        return f"Reminder: Post-mortem for incident #{self.incident.id} is still pending. :memo:"
+
+    def get_blocks(self) -> list[Block]:
+        return [
+            SectionBlock(
+                text=f":hourglass_flowing_sand: *Reminder* :hourglass_flowing_sand:\n"
+                f"The post-mortem for incident *#{self.incident.id}* is still pending. Completing it will "
+                f"help us understand and mitigate such issues in the future. :memo:"
+            ),
+            ContextBlock(
+                elements=[
+                    TextObject(
+                        type="mrkdwn",
+                        text=":bulb: Please take a moment to complete the post-mortem process. "
+                        "It’s essential for continuous improvement!",
+                    )
+                ]
+            ),
+        ]
+
+
+class SlackMessageIncidentUpdateReminderCommander(SlackMessageSurface):
+    id = "ff_incident_reminder_commander"
+
+    def __init__(self, incident: Incident, time_delta_fmt: str):
+        self.incident = incident
+        self.time_delta_fmt = time_delta_fmt
+        super().__init__()
+
+    def get_blocks(self) -> list[Block]:
+        if self.incident.priority.value >= 4:
+            text = f"Hey there ! You may want to close your this {self.incident.slack_channel_name} incident 📝"
+        else:
+            text = f"It is a {self.incident.slack_channel_name} incident and a lot of people, customers and employees, are probably impacted. This incident has no update for {self.time_delta_fmt}. Please make sure it is done even if you are not the one doing it."
```

Comment:
Please review text, `It is a #20240101-1234-internal incident` does not mean anything.